recipes/tagine.md

```diff
@@ -0,0 +1,53 @@
+# Lubes' Vegetable Tagine
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-60%25-orange.svg) 
+![Speed Badge](https://img.shields.io/badge/Speed-2hr-orange.svg)
+
+
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
```

Comment:
[Honey ain't vegan pal](https://www.vegansociety.com/go-vegan/why-honey-not-vegan)